packages/coinstac-ui/app/render/index.js

```diff
@@ -28,10 +27,9 @@ app.analysisRequestId = app.analysisRequestId || 0;
 require('./styles/app.scss');
 
 const rootEl = document.getElementById('app');
-const store = configureStore();
 
 render(
-  <Root history={hashHistory} store={store} />,
```

Comment:
Why this change? Seems like this will break hot reloading as the store and history are created anew in `Root.jsx` versus here, where they were created in a higher closure for reuse for Webpack’s hot reloading machinery.